elastic/kibana
# [Infra][Hosts] Override tooltip visibility for syncing multiple metrics charts

## Summary

This PR provides the necessary configuration to have the lens charts on the Infra -> Hosts View page to have their tooltips synchronised when one hovers the mouse over a chart. As one hovers over the datapoints in one chart, the other charts will show their tooltips synchronised to the same timepoint.

https://github.com/user-attachments/assets/5005cfc3-a739-4c87-8800-e4f3af2c0717


### How to test

- Go to Infrastructure -> Hosts page
- Scroll to metrics charts once page is loaded
- Mouse over a chart across various data points

**Expected**: All charts should show a tooltip at the same time instead of just the chart being hovered over.

Closes #152102
